api/rbac/v1alpha1/role_types.go

```diff
@@ -32,8 +30,11 @@ type ResourceDetails struct {
 	Verbs        []string `json:"verbs,omitempty" protobuf:"bytes,3,rep,name=verbs"`
 }
 
+type Claim struct {
+	Name   string
+	Values []string
+}
```

Comment:
Are struct tags missing? It seems like they should be here.

Reply:
This was removed from the suggestion below.